sdrangel: build fail

### Nixpkgs version

- Unstable (25.05)

### Describe the bug

sdrangel 7.22.6 fails to build

### Steps to reproduce

update nixpkgs-unstable, rebuild the system

### Expected behaviour

build successfully complete

### Screenshots

_No response_

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/jl3rvx7rxzzvnsfjhxcqk5qydjvjm94q-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/share/doc/sdrangel -DCMAKE_INSTALL_INFODIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/include -DCMAKE_INSTALL_SBINDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/hqbi26nfw5ag6jwjbjxw6gnn95kk7454-sdrangel-7.22.6 -DAPT_DIR=/nix/store/2qph6x9fvydv90gxjhxyrv9f473lq6ij-aptdec-1.8.0 -DDAB_DIR=/nix/store/3cvi43xmznlbi3isaqllrbh2rfzx9xnm-dab_lib-unstable-2023-03-02 -DSGP4_DIR=/nix/store/6sb182aq9if2v16wp91vq403cyd0dl86-sgp4-unstable-2022-11-13 -DSOAPYSDR_DIR=/nix/store/4dmd5b8zqlz3mmzg7363xj5i5y6pipqk-soapysdr-0.8.1-unstable-2025-03-30-03 -Wno-dev -DENABLE_QT6=ON -DQT_OPTIONAL_TOOLS_PATH=/nix/store/sxw28a1dc1cbksn2hyrg8g7m5lylr1vb-qttools-6.9.0 -DQT_OPTIONAL_TOOLS_PATH=/nix/store/sxw28a1dc1cbksn2hyrg8g7m5lylr1vb-qttools-6.9.0 -DPROTOC_EXE=/nix/store/s0521fkq7z9lycdgal6y7g726vz5d6zk-protobuf-29.4/bin/protoc -DProtobuf_PROTOC_EXE=/nix/store/s0521fkq7z9lycdgal6y7g726vz5d6zk-protobuf-29.4/bin/protoc -DProtobuf_PROTOC_EXECUTABLE=/nix/store/s0521fkq7z9lycdgal6y7g726vz5d6zk-protobuf-29.4/bin/protoc
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
-- SDRangel Version: 7.22.6
-- Build type set to: Release
-- Compiling for 24 bit Rx DSP chain
-- Looking for __x86_64__
-- Looking for __x86_64__ - found
-- Target architecture: x86_64-native
-- Performing Test FLAG_SUPPORTED
-- Performing Test FLAG_SUPPORTED - Success
-- Looking for __ARM_NEON
-- Looking for __ARM_NEON - not found
-- Looking for __AVX512F__
-- Looking for __AVX512F__ - not found
-- Looking for __AVX2__
-- Looking for __AVX2__ - not found
-- Looking for __AVX__
-- Looking for __AVX__ - not found
-- Looking for __SSE4_2__
-- Looking for __SSE4_2__ - not found
-- Looking for __SSE4_1__
-- Looking for __SSE4_1__ - not found
-- Looking for __SSSE3__
-- Looking for __SSSE3__ - not found
-- Looking for __SSE3__
-- Looking for __SSE3__ - not found
-- Looking for __SSE2__
-- Looking for __SSE2__ - found
-- Interprocedural optimization enabled
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/sq50878z0rdxwdyjw3fvbgf6wkpcnpcp-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/xwhkngnns6ax8fgl7igdd6pqwkav9n5b-vulkan-headers-1.4.309.0/include
-- Found Cups: /nix/store/y7gsczfhb18x1pvzhzzz9hw89vdqncck-cups-2.4.11-lib/lib/libcups.so (found version "2.4.11")
-- Found PkgConfig: /nix/store/bbp56ai8w43xkfisar83h37imjdlcx6r-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Found Boost: /nix/store/ijdpzk8xcsq77k4g4fy19wr88d6s8pmg-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found version "1.87.0")
-- Checking for module 'fftw3f >= 3.0'
--   Found fftw3f, version 3.3.10
-- Found FFTW3F: /nix/store/fjq3gffhn4pv2fvbyh344s6a6m72mmh6-fftw-single-3.3.10/lib/libfftw3f.so
-- Found FFTW3F: /nix/store/62b5a77fj2i7h3mr7wwyaj12q6rklgh0-fftw-single-3.3.10-dev/include, /nix/store/fjq3gffhn4pv2fvbyh344s6a6m72mmh6-fftw-single-3.3.10/lib/libfftw3f.so, /nix/store/fjq3gffhn4pv2fvbyh344s6a6m72mmh6-fftw-single-3.3.10/lib/libfftw3f_threads.so
-- Checking for module 'libusb-1.0'
--   Found libusb-1.0, version 1.0.28
-- Found libusb-1.0: /nix/store/2474w2chiwbxaalxin0m0hdhf18nb6zq-libusb-1.0.28-dev/include/libusb-1.0, /nix/store/g8ydxqj2dhvar6crwbv8q3k5grjjnwvr-libusb-1.0.28/lib/libusb-1.0.so
-- Found OpenCV: / (found version "4.11.0") found components: core highgui imgproc imgcodecs videoio
-- Checking for module 'libsigmf'
--   No package 'libsigmf' found
-- libsigmf not found.
-- Found ZLIB: /nix/store/jk27jlxfsljcmz4jgxkxkmrc1hjgmq1p-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Checking for module 'faad2'
--   Found faad2, version 2.11.2
-- FAAD LIBRARY /nix/store/w613kcix2rx6p72ym0pkg97jy47clh7v-faad2-2.11.2/lib/libfaad.so
-- FAAD INCLUDE DIRS /nix/store/ps1r2my6rz9msfxb7m6nm1v123gqycz6-faad2-2.11.2-dev/include
-- Found FAAD: /nix/store/w613kcix2rx6p72ym0pkg97jy47clh7v-faad2-2.11.2/lib/libfaad.so
-- Checking for module 'codec2'
--   Found codec2, version 1.2.0
-- Found Codec2: /nix/store/2rdc8lnxp3ilq1p1cbv751wk9yqzh6rb-codec2-1.2.0/include, /nix/store/2rdc8lnxp3ilq1p1cbv751wk9yqzh6rb-codec2-1.2.0/lib/libcodec2.so
-- Found CODEC2: /nix/store/2rdc8lnxp3ilq1p1cbv751wk9yqzh6rb-codec2-1.2.0/lib/libcodec2.so
-- Checking for module 'libcm256cc'
--   Found libcm256cc, version 1.1.0
-- Found CM256cc: /nix/store/2i0mvangvvflc8skwbbya754rpchn5gj-cm256cc-1.1.0/include, /nix/store/2i0mvangvvflc8skwbbya754rpchn5gj-cm256cc-1.1.0/lib/libcm256cc.so
-- Found CM256CC: /nix/store/2i0mvangvvflc8skwbbya754rpchn5gj-cm256cc-1.1.0/lib/libcm256cc.so
-- Found LibMbe: /nix/store/gdcxpc9z99q666dpqga1l6chx5akdj5v-mbelib-1.3.0/include, 
-- Checking for module 'libdsdcc'
--   Found libdsdcc, version 1.9.0
-- Found libdsdcc: /nix/store/d6sg058adlp8jsn1z6vq4rjjxx5ppd9q-dsdcc-1.9.5/include, /nix/store/d6sg058adlp8jsn1z6vq4rjjxx5ppd9q-dsdcc-1.9.5/lib/libdsdcc.so
-- Checking for module 'sgp4'
--   No package 'sgp4' found
-- SGP4 LIBRARIES /nix/store/6sb182aq9if2v16wp91vq403cyd0dl86-sgp4-unstable-2022-11-13/lib/libsgp4s.so
-- SGP4 INCLUDE DIRS /nix/store/6sb182aq9if2v16wp91vq403cyd0dl86-sgp4-unstable-2022-11-13/include/libsgp4
-- Found SGP4: /nix/store/6sb182aq9if2v16wp91vq403cyd0dl86-sgp4-unstable-2022-11-13/lib/libsgp4s.so
-- Checking for module 'apt'
--   No package 'apt' found
-- APT LIBRARIES /nix/store/2qph6x9fvydv90gxjhxyrv9f473lq6ij-aptdec-1.8.0/lib/libapt.so
-- APT INCLUDE DIRS /nix/store/2qph6x9fvydv90gxjhxyrv9f473lq6ij-aptdec-1.8.0/include/apt
-- Found APT: /nix/store/2qph6x9fvydv90gxjhxyrv9f473lq6ij-aptdec-1.8.0/lib/libapt.so
-- Checking for module 'libdab'
--   No package 'libdab' found
-- Found libdab: /nix/store/3cvi43xmznlbi3isaqllrbh2rfzx9xnm-dab_lib-unstable-2023-03-02/include/dab_lib, /nix/store/3cvi43xmznlbi3isaqllrbh2rfzx9xnm-dab_lib-unstable-2023-03-02/lib/libdab_lib.so
-- Found HIDAPI: /nix/store/bq39wcsfz4pz1azqb2hi99s4n9xybgxb-hidapi-0.14.0/include/hidapi  found components: any
-- Checking for module 'libavcodec'
--   Found libavcodec, version 61.19.101
-- Checking for module 'libavformat'
--   Found libavformat, version 61.7.100
-- Checking for module 'libavdevice'
--   Found libavdevice, version 61.3.100
-- Checking for module 'libavutil'
--   Found libavutil, version 59.39.100
-- Checking for module 'libavfilter'
--   Found libavfilter, version 10.4.100
-- Checking for module 'libswscale'
--   Found libswscale, version 8.3.100
-- Checking for module 'libpostproc'
--   Found libpostproc, version 58.3.100
-- Checking for module 'libswresample'
--   Found libswresample, version 5.3.100
-- Found FFmpeg: /nix/store/8mi2n0ir1i4qkdcgbm3m6i5bfax26xxq-ffmpeg-7.1.1-lib/lib/libavcodec.so;/nix/store/8mi2n0ir1i4qkdcgbm3m6i5bfax26xxq-ffmpeg-7.1.1-lib/lib/libavformat.so;/nix/store/8mi2n0ir1i4qkdcgbm3m6i5bfax26xxq-ffmpeg-7.1.1-lib/lib/libavutil.so;/nix/store/8mi2n0ir1i4qkdcgbm3m6i5bfax26xxq-ffmpeg-7.1.1-lib/lib/libswscale.so
-- Checking for module 'libggmorse'
--   No package 'libggmorse' found
-- GGMorse not found
-- Could NOT find GGMORSE (missing: GGMORSE_LIBRARIES GGMORSE_INCLUDE_DIR) 
-- Checking for module 'libairspy'
--   Found libairspy, version 1.0
-- Found libairspy: /nix/store/km4avwv29yardz24jaihdgr2zv8hpnry-airspy-1.0.10/include, /nix/store/km4avwv29yardz24jaihdgr2zv8hpnry-airspy-1.0.10/lib/libairspy.so
-- Checking for module 'libairspyhf'
--   Found libairspyhf, version 1.6
-- Found libairspyhf: /nix/store/kcvmkdifya2p28i2wb0wwms8bxmqczh2-airspyhf-1.6.8/include, /nix/store/kcvmkdifya2p28i2wb0wwms8bxmqczh2-airspyhf-1.6.8/lib/libairspyhf.so
-- Checking for module 'libbladeRF>=2.0'
--   Found libbladeRF, version 2.5.0-git-unknown
-- Found libbladerf: /nix/store/jsbz5bhq2bxqsh0ml87rp79lxywlppxp-libbladeRF-2.5.0/include, /nix/store/jsbz5bhq2bxqsh0ml87rp79lxywlppxp-libbladeRF-2.5.0/lib/libbladeRF.so
-- Checking for module 'libhackrf'
--   Found libhackrf, version 0.9
-- Found libhackrf: /nix/store/r746lakyvma5k6prv3w8iwbzpfd45z7g-hackrf-2024.02.1/include, /nix/store/r746lakyvma5k6prv3w8iwbzpfd45z7g-hackrf-2024.02.1/lib/libhackrf.so
-- Found Lime Suite: /nix/store/jya8x95mq19wcnffdays101z29syngcq-limesuite-23.11.0/include, /nix/store/jya8x95mq19wcnffdays101z29syngcq-limesuite-23.11.0/lib/libLimeSuite.so
-- Checking for module 'libiio>=0.7'
--   Found libiio, version 0.24
-- Found libiio: /nix/store/0c8v2cjrrkc3r9c85shbksqldpy44aa0-libiio-0.24-dev/include, /nix/store/gna2lw3zwllvq1a7qw6xip1aw10liydi-libiio-0.24-lib/lib/libiio.so
-- Checking for module 'libmirisdr'
--   No package 'libmirisdr' found
-- libmirisdr not found.
-- Checking for module 'libperseus'
--   No package 'libperseus' found
-- libperseus not found.
-- Checking for module 'librtlsdr'
--   Found librtlsdr, version 0.6.5
-- Found librtlsdr: /nix/store/hifyrs0l8bydgxk0wwnlxcghgkdwnw1c-rtl-sdr-blog-1.3.5/include, /nix/store/hifyrs0l8bydgxk0wwnlxcghgkdwnw1c-rtl-sdr-blog-1.3.5/lib/librtlsdr.so
-- Checking for one of the modules 'SoapySDR>=0.4.0'
-- Found SoapySDR: version 0.8-3, /nix/store/4dmd5b8zqlz3mmzg7363xj5i5y6pipqk-soapysdr-0.8.1-unstable-2025-03-30-03, /nix/store/4dmd5b8zqlz3mmzg7363xj5i5y6pipqk-soapysdr-0.8.1-unstable-2025-03-30-03/include, /nix/store/4dmd5b8zqlz3mmzg7363xj5i5y6pipqk-soapysdr-0.8.1-unstable-2025-03-30-03/lib/libSoapySDR.so
-- Checking for module 'uhd'
--   Found uhd, version 4.7.0.0-0-unknown
-- UHD LIBRARIES /nix/store/clr7n16chgfvfcv06zzlr8p7kgqsbcl5-uhd-4.7.0.0/lib/libuhd.so
-- UHD INCLUDE DIRS /nix/store/0wdjky54n22mq7w0bv7jp6wsghq64dhl-uhd-4.7.0.0-dev/include
-- Found UHD: /nix/store/clr7n16chgfvfcv06zzlr8p7kgqsbcl5-uhd-4.7.0.0/lib/libuhd.so
-- SDRPLAY LIBRARIES SDRPLAY_LIBRARIES-NOTFOUND
-- SDRPLAY INCLUDE DIRS SDRPLAY_INCLUDE_DIR-NOTFOUND
-- Could NOT find SDRPLAY (missing: SDRPLAY_LIBRARIES SDRPLAY_INCLUDE_DIR) 
-- Checking for module 'libxtrx'
--   No package 'libxtrx' found
-- libxtrx not found.
-- Found Opus: /nix/store/mlk4l6s5yl7i7c4r1fl8m15vhhsz3fg6-libopus-1.5.2-dev/include
-- Could not find nvcc, please set CUDAToolkit_ROOT.
-- Not building remotesink (ENABLE_CHANNELRX_REMOTESINK=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sigmffilesink (ENABLE_CHANNELRX_SIGMFFILESINK=ON LIBSIGMF_FOUND=FALSE)
-- Include demoddatv
-- Not building remotesource (ENABLE_CHANNELTX_REMOTESOURCE=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building ambe (ENABLE_FEATURE_AMBE=ON LIBSERIALDV_FOUND=FALSE)
-- Not building morsedecoder (ENABLE_FEATURE_MORSEDECODER=ON GGMODSE_FOUND=FALSE)
-- Not building xtrxmimo (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Checking for module 'hamlib'
--   No package 'hamlib' found
-- Hamlib not found
-- Could NOT find HAMLIB (missing: hamlib_LIBRARIES hamlib_INCLUDE_DIRS) 
-- Not building audiocatsiso (hamlib_FOUND=FALSE)
-- Not building remoteinput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sdrplay (ENABLE_MIRISDR=ON LIBMIRISDR_FOUND=FALSE)
-- Not building sdrplayv3 (ENABLE_SDRPLAY=ON SDRPLAY_FOUND=FALSE)
-- Not building perseus (ENABLE_PERSEUS=ON LIBPERSEUS_FOUND=FALSE RX_SAMPLE_24BIT=ON)
-- Not building xtrxinput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building sigmffileinput (LIBSIGMF_FOUND=FALSE)
-- Not building remoteoutput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building xtrxoutput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building remotesink (ENABLE_CHANNELRX_REMOTESINK=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sigmffilesink (ENABLE_CHANNELRX_SIGMFFILESINK=ON LIBSIGMF_FOUND=FALSE)
-- Not building remotesource (ENABLE_CHANNELTX_REMOTESOURCE=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building jogdialcontroller (ENABLE_FEATURE_JOGDIALCONTROLLER=ON SERVER_MODE=ON)
-- Not building ambe (ENABLE_FEATURE_AMBE=ON LIBSERIALDV_FOUND=FALSE)
-- Not building morsedecoder (ENABLE_FEATURE_MORSEDECODER=ON GGMODSE_FOUND=FALSE)
-- Not building xtrxmimo (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Checking for module 'hamlib'
--   No package 'hamlib' found
-- Hamlib not found
-- Could NOT find HAMLIB (missing: hamlib_LIBRARIES hamlib_INCLUDE_DIRS) 
-- Not building audiocatsiso (hamlib_FOUND=FALSE)
-- Not building remoteinput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sdrplay (ENABLE_MIRISDR=ON LIBMIRISDR_FOUND=FALSE)
-- Not building sdrplayv3 (ENABLE_SDRPLAY=ON SDRPLAY_FOUND=FALSE)
-- Not building perseus (ENABLE_PERSEUS=ON LIBPERSEUS_FOUND=FALSE RX_SAMPLE_24BIT=ON)
-- Not building xtrxinput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building sigmffileinput (LIBSIGMF_FOUND=FALSE)
-- Not building remoteoutput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building xtrxoutput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Doxygen not found, developer documentation can not be generated
-- Asking qmake for QT_PLUGINS_DIR and got /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/lib/qt-6/plugins
-- Asking qmake for QT_IMPORTS_DIR and got **Unknown**
-- Asking qmake for QT_QML_DIR and got /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/lib/qt-6/qml
-- Asking qmake for QT_LIBS_DIR and got /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/lib
-- CPack generators: 
-- Configuring done (13.8s)
-- Generating done (2.5s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    PROTOC_EXE
    Protobuf_PROTOC_EXE
    Protobuf_PROTOC_EXECUTABLE

[0m
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j16
[1/3725] Automatic MOC and UIC for target fcdhid
[2/3725] Automatic MOC and UIC for target modemm17
[3/3725] Automatic MOC and UIC for target ldpctool
[4/3725] Automatic MOC and UIC for target fcdlib
[5/3725] Building C object fcdhid/CMakeFiles/fcdhid.dir/__/custom/apple/apple_compat.c.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[6/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/modemm17_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[7/3725] Automatic MOC and UIC for target wdsp
[8/3725] Building CXX object fcdhid/CMakeFiles/fcdhid.dir/fcdhid_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[9/3725] Building CXX object fcdlib/CMakeFiles/fcdlib.dir/fcdlib_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[10/3725] Building CXX object fcdlib/CMakeFiles/fcdlib.dir/fcdtraits.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[11/3725] Building CXX object wdsp/CMakeFiles/wdsp.dir/wdsp_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[12/3725] Building C object wdsp/CMakeFiles/wdsp.dir/__/custom/apple/apple_compat.c.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[13/3725] Building C object fcdhid/CMakeFiles/fcdhid.dir/fcdhid.c.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[14/3725] Automatic MOC and UIC for target qrtplib
[15/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/ClockRecovery.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[16/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/M17Randomizer.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[17/3725] Automatic MOC and UIC for target logging
[18/3725] Automatic MOC and UIC for target httpserver
[19/3725] Building CXX object fcdlib/CMakeFiles/fcdlib.dir/fcdproplusconst.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[20/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/FreqDevEstimator.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[21/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/Golay24.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[22/3725] Building CXX object fcdlib/CMakeFiles/fcdlib.dir/fcdproconst.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[23/3725] Building CXX object wdsp/CMakeFiles/wdsp.dir/amd.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[24/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/LinkSetupFrame.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[25/3725] Building CXX object wdsp/CMakeFiles/wdsp.dir/ammod.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[26/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/Correlator.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[27/3725] Automatic MOC and UIC for target swagger
[28/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpglobal.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[29/3725] Building CXX object qrtplib/CMakeFiles/qrtplib.dir/rtcpapppacket.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[30/3725] Building CXX object qrtplib/CMakeFiles/qrtplib.dir/rtcpbyepacket.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[31/3725] Building CXX object qrtplib/CMakeFiles/qrtplib.dir/qrtplib_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[32/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpcookie.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[33/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/M17Modulator.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
In file included from /build/source/modemm17/M17Modulator.h:6,
                 from /build/source/modemm17/M17Modulator.cpp:1:
/build/source/modemm17/FirFilter.h: In instantiation of 'modemm17::BaseFirFilter<N> modemm17::makeFirFilter(const std::array<float, N>&) [with long unsigned int N = 150]':
/build/source/modemm17/M17Modulator.h:458:26:   required from here
  458 |         rrc(makeFirFilter(rrc_taps)),
      |             ~~~~~~~~~~~~~^~~~~~~~~~
/build/source/modemm17/FirFilter.h:54:48: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
   54 |         return std::move(BaseFirFilter<N>(taps));
      |                                                ^
/build/source/modemm17/FirFilter.h:54:48: note: remove 'std::move' call
[34/3725] Building CXX object modemm17/CMakeFiles/modemm17.dir/M17Demodulator.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[35/3725] Building CXX object logging/CMakeFiles/logging.dir/logging_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[36/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpsession.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[37/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpresponse.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[38/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httprequesthandler.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[39/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httplistener.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[40/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpserver_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[41/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpconnectionhandler.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[42/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpconnectionhandlerpool.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
/build/source/httpserver/httpconnectionhandlerpool.cpp: In member function 'void qtwebapp::HttpConnectionHandlerPool::loadSslConfig()':
/build/source/httpserver/httpconnectionhandlerpool.cpp:161:49: warning: 'QSsl::TlsV1_0' is deprecated: Use TlsV1_2OrLater instead. [-Wdeprecated-declarations]
  161 |             sslConfiguration->setProtocol(QSsl::TlsV1_0);
      |                                                 ^~~~~~~
In file included from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork/qsslcertificate.h:19,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork/qsslerror.h:11,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork/qsslsocket.h:12,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork/QSslSocket:1,
                 from /build/source/httpserver/httpconnectionhandlerpool.cpp:2:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork/qssl.h:51:9: note: declared here
   51 |         TlsV1_0 QT_DEPRECATED_VERSION_X_6_3("Use TlsV1_2OrLater instead."),
      |         ^~~~~~~
[43/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httprequest.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[44/3725] Building CXX object httpserver/CMakeFiles/httpserver.dir/httpsessionstore.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[45/3725] Building CXX object qrtplib/CMakeFiles/qrtplib.dir/rtcpsdesinfo.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set

[...]

warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2015/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodsink.cpp.o
FAILED: plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodsink.cpp.o 
/nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/g++ -DAPPLICATION_NAME=\"SDRangel\" -DARCHITECTURE_x86_64 -DBUILD_SHARED_LIBS -DCOMPANY=\"f4exb\" -DCOPYRIGHT="\"Copyright (c) 2019 Edouard Griffiths. All rights reserved.\"" -DHAS_FT8 -DHAS_LIMERFEUSB -DHAS_WDSP -DIDENTIFIER=\"com.f4exb.SDRangel\" -DLIB=\"lib\" -DLINUX -DQT_CHARTS_FOUND -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_GUI_LIB -DQT_LOCATION_FOUND -DQT_MULTIMEDIA_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_NO_DEBUG_OUTPUT -DQT_OPENGLWIDGETS_LIB -DQT_OPENGL_LIB -DQT_PLUGIN -DQT_POSITIONING_FOUND -DQT_POSITIONING_LIB -DQT_STATEMACHINE_LIB -DQT_TEXTTOSPEECH_FOUND -DQT_WEBSOCKETS_LIB -DQT_WIDGETS_LIB -DSDRANGEL_LIB_VERSION=\"7.22.6\" -DSDRANGEL_VERSION=\"7.22.6\" -DSDR_RX_SAMPLE_24BIT -DUSE_SSE2 -Ddemodnavtex_EXPORTS -I/build/source/build/plugins/channelrx/demodnavtex -I/build/source/plugins/channelrx/demodnavtex -I/build/source/build/plugins/channelrx/demodnavtex/demodnavtex_autogen/include -I/build/source/sdrbase -I/build/source/exports -I/build/source/sdrgui -I/build/source/swagger/sdrangel/code/qt5/client -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/mkspecs/linux-g++ -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtGui -isystem /nix/store/blim4rjiay9856ib3xqycfzrq2hrmrip-qtmultimedia-6.9.0/include/QtMultimedia -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtNetwork -isystem /nix/store/w987h4wjzvqs0vv9s7m71zmawvcplvn6-qtwebsockets-6.9.0/include/QtWebSockets -isystem /nix/store/xp7prx5mm8zfrj6m25lx01xcw2dgib2y-qtpositioning-6.9.0/include/QtPositioning -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/6.9.0 -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/6.9.0/QtCore -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtOpenGLWidgets -isystem /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtOpenGL -isystem /nix/store/291n1j6ychkvr1y6h0aj2f97jsgy6b17-qtscxml-6.9.0/include/QtStateMachine -O3 -DNDEBUG -std=c++17 -flto=auto -fno-fat-lto-objects -fPIC -march=native -Wall -Wextra -Wvla -ffast-math -fno-finite-math-only -ftree-vectorize -Woverloaded-virtual -fPIC -MD -MT plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodsink.cpp.o -MF plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodsink.cpp.o.d -o plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodsink.cpp.o -c /build/source/plugins/channelrx/demodnavtex/navtexdemodsink.cpp
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
/build/source/plugins/channelrx/demodnavtex/navtexdemodsink.cpp: In member function 'void NavtexDemodSink::receiveBit(bool)':
/build/source/plugins/channelrx/demodnavtex/navtexdemodsink.cpp:357:51: error: no matching function for call to 'QChar::QChar(signed char&)'
  357 |                     m_messageBuffer.append(QChar(c));
      |                                                   ^
In file included from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qstring.h:14,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qhashfunctions.h:9,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qfloat16.h:10,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qdebug.h:14,
                 from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/QDebug:1,
                 from /build/source/plugins/channelrx/demodnavtex/navtexdemodsink.cpp:19:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:149:43: note: candidate: 'template<class Char, typename std::enable_if<std::is_same<Char, unsigned char>::value, bool>::type <anonymous> > constexpr QChar::QChar(Char)'
  149 |     QT_ASCII_CAST_WARN constexpr explicit QChar(const Char c) noexcept : ucs(c) { }
      |                                           ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:149:43: note:   template argument deduction/substitution failed:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:148:66: error: no type named 'type' in 'struct std::enable_if<false, bool>'
  148 |     template <typename Char, if_explicit_ascii_warn_char<Char> = true>
      |                                                                  ^~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:143:45: note: candidate: 'template<class Char, typename std::enable_if<std::is_same<Char, char>::value, bool>::type <anonymous> > constexpr QChar::QChar(Char)'
  143 |     QT_ASCII_CAST_WARN constexpr Q_IMPLICIT QChar(const Char ch) noexcept : ucs(uchar(ch)) {}
      |                                             ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:143:45: note:   template argument deduction/substitution failed:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:142:66: error: no type named 'type' in 'struct std::enable_if<false, bool>'
  142 |     template <typename Char, if_implicit_ascii_warn_char<Char> = true>
      |                                                                  ^~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:134:24: note: candidate: 'template<class Char, typename std::enable_if<std::disjunction<std::is_same<Char, char32_t>, std::conjunction<std::is_same<Char, wchar_t>, std::integral_constant<bool, true> >, std::is_same<Char, int>, std::is_same<Char, unsigned int> >::value, bool>::type <anonymous> > constexpr QChar::QChar(Char)'
  134 |     constexpr explicit QChar(const Char ch) noexcept
      |                        ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:134:24: note:   template argument deduction/substitution failed:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:133:66: error: no type named 'type' in 'struct std::enable_if<false, bool>'
  133 |     template <typename Char, if_explicit_conversion_char<Char> = true>
      |                                                                  ^~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:132:26: note: candidate: 'template<class Char, typename std::enable_if<std::disjunction<std::is_same<Char, short unsigned int>, std::is_same<Char, short int>, std::is_same<Char, QChar::SpecialCharacter>, std::is_same<Char, QLatin1Char>, std::conjunction<std::is_same<Char, wchar_t>, std::negation<std::integral_constant<bool, true> > >, std::is_same<Char, char16_t> >::value, bool>::type <anonymous> > constexpr QChar::QChar(Char)'
  132 |     constexpr Q_IMPLICIT QChar(const Char ch) noexcept : ucs(char16_t(ch)) {}
      |                          ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:132:26: note:   template argument deduction/substitution failed:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:131:66: error: no type named 'type' in 'struct std::enable_if<false, bool>'
  131 |     template <typename Char, if_implicit_conversion_char<Char> = true>
      |                                                                  ^~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:119:24: note: candidate: 'constexpr QChar::QChar(uchar, uchar)'
  119 |     constexpr explicit QChar(uchar c, uchar r) noexcept : ucs(char16_t((r << 8) | c)) {}
      |                        ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:119:24: note:   candidate expects 2 arguments, 1 provided
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:115:26: note: candidate: 'constexpr QChar::QChar()'
  115 |     constexpr Q_IMPLICIT QChar() noexcept : ucs(0) {}
      |                          ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:115:26: note:   candidate expects 0 arguments, 1 provided
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:48:31: note: candidate: 'constexpr QChar::QChar(const QChar&)'
   48 | class QT6_ONLY(Q_CORE_EXPORT) QChar {
      |                               ^~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:48:31: note:   no known conversion for argument 1 from 'signed char' to 'const QChar&'
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:48:31: note: candidate: 'constexpr QChar::QChar(QChar&&)'
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtCore/qchar.h:48:31: note:   no known conversion for argument 1 from 'signed char' to 'QChar&&'
[2016/3725] Building CXX object plugins/channelrx/demodft8/CMakeFiles/demodft8.dir/ft8demodsettingsdialog.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2017/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/demodnavtex_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2018/3725] Building CXX object plugins/channelrx/noisefigure/CMakeFiles/noisefigure.dir/noisefigureenrdialog.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2019/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodplugin.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2020/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemod.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2021/3725] Building CXX object plugins/channelrx/noisefigure/CMakeFiles/noisefigure.dir/noisefiguregui.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2022/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodwebapiadapter.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2023/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/rttydemodsettings.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2024/3725] Building CXX object plugins/channelrx/demodft8/CMakeFiles/demodft8.dir/ft8demodgui.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2025/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/rttydemodbaseband.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2026/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/demodrtty_autogen/mocs_compilation.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2027/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/rttydemodsink.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2028/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/rttydemod.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2029/3725] Building CXX object plugins/channelrx/demodrtty/CMakeFiles/demodrtty.dir/rttydemodplugin.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
[2030/3725] Building CXX object plugins/channelrx/demodnavtex/CMakeFiles/demodnavtex.dir/navtexdemodgui.cpp.o
warning: Skipping impure flag -march=native because NIX_ENFORCE_NO_NATIVE is set
ninja: build stopped: subcommand failed.
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.13.10, NixOS, 25.05 (Warbler), 25.05.20250408.c8cd814`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.14`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/m7y9g8z15va6sd0h12zc3iw0acxn7463-source`

### Notify maintainers

@alkeryn @Tungsten842

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+bug%22+-label%3A%226.topic%3A+darwin%22+-label%3A%226.topic%3A+nixos%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/radio/sdrangel/default.nix
+++ b/pkgs/applications/radio/sdrangel/default.nix
@@ -12,6 +12,7 @@
   dsdcc,
   faad2,
   fetchFromGitHub,
+  fetchpatch,
   fftwFloat,
   flac,
   glew,
@@ -114,6 +115,14 @@ stdenv.mkDerivation (finalAttrs: {
     ]
     ++ lib.optionals stdenv.hostPlatform.isLinux [ qtwayland ]
     ++ lib.optionals withSDRplay [ sdrplay ];
+
+  patches = [
+    # https://github.com/f4exb/sdrangel/pull/2439
+    (fetchpatch {
+      url = "https://github.com/f4exb/sdrangel/commit/60869b74f96b26e8a173f3f215c2badeaef9a136.patch";
+      hash = "sha256-Lq9pyissNmLYavLCISga0EWbRwisGnKiz6UYhzxJatc=";
+    })
+  ];
 
   cmakeFlags = [
     "-DAPT_DIR=${aptdec}"

